src/components/__test__/Home.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../redux/configStore';
+import Home from '../Home';
+
+describe('Home test', () => {
+  it('renders leagues', () => {
+    const app = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Home />
+        </BrowserRouter>
+      </Provider>,
+    )
+      .toJSON();
+    expect(app).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Great work using the renderer to test the components by checking the snapshot. To complete the tests for this project based on the requirements, kindly do the following:

- Write unit tests using Jest for some pure functions in your redux code
- Write integration tests for your React components using React Testing Library. Kindly read this [article](https://www.freecodecamp.org/news/8-simple-steps-to-start-testing-react-apps-using-react-testing-library-and-jest/) as a guide. 👍

Reply:
Hi @Gambit142, Thank you for reviewing my PR